batch/batch/front_end/front_end.py

```diff
@@ -964,8 +965,7 @@ async def close_batch(request, userdata):
                 reason=f'wrong number of jobs: expected {expected_n_jobs}, actual {actual_n_jobs}')
         raise
 
-    async with in_cluster_ssl_client_session(
-            raise_for_status=True, timeout=aiohttp.ClientTimeout(total=60)) as session:
```

Comment:
This seemed like too generous a timeout, so I'm reverting it to the default of 5s.